docs/conf.py

```diff
@@ -38,6 +38,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.viewcode',
     'sphinx.ext.githubpages',
+    'qcodes.sphinx_extensions.add_parameter',
```

Comment:
let's make this dependency optional - contrib drivers does not restrict the qcodes version very much, hence it as to be flexible. so the guard could be ``try-except-ImportError`` or explicitly checkin for qcodes version > X.